packages/front-end/pages/settings/index.tsx

```diff
@@ -223,6 +223,8 @@ export const supportedCurrencies = {
   ZWL: "Zimbabwe Dollar (ZWL)",
 };
 
+export const DEFAULT_SRM_THRESHOLD = 0.001;
```

Comment:
Considered adding this to `shared/constants` but my understanding is this value will only be used by the front-end so I place it here